src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = days * 40;
+  const discountForThreeDaysOrMore = days >= 3 && days < 7;
+  const discountForSevenDaysOrMore = days >= 7;
+
+  if (discountForThreeDaysOrMore) {
+    return price - 20;
+  };
+
+  if (discountForSevenDaysOrMore) {
+    return price - 50;
```

Comment:
pass this magic `20` and `50` into variables too

Reply:
Didn't fix, maybe you forgot to push your changes

Reply:
Still not fixed(